docs/sumbitting-bulk-sentences.md

```diff
@@ -0,0 +1,67 @@
+All the sentences read by our contributors are sourced from copyright free sources, either through automated sourcing from known sources like Wikipedia or contributed by our language communities.
+
+If you want to add a single sentence, or a series of single sentences, you can do so via the [Sentence Collector](link to new blog post). But when you have a large volume of sentences to contribute, you can make a bulk sentence submission using GitHub.
+
+Remember that for both single sentence and bulk sentence submissions, sentences must:
+- Be in the public domain, with a CC0 license
+- Be short, readable and take about 10-15 seconds to read
+- Avoid including numbers or special characters
+
+Please format your bulk sentences into a TSV file with two columns, one containing the sentences you would like to submit and the other containing the source of the sentences.
```

Comment:
Are we using TSV or CSV files? I remember seeing something about CSV files somewhere